multitenancy/src/main/java/org/axonframework/extensions/multitenancy/configuration/MultiTenantEventProcessingModule.java

```diff
@@ -60,19 +72,39 @@ public class MultiTenantEventProcessingModule extends EventProcessingModule {
      */
     public MultiTenantEventProcessingModule(TenantProvider tenantProvider) {
         this.tenantProvider = tenantProvider;
+        this.multiTenantDeadLetterQueueFactory = null;
         multiTenantStreamableMessageSourceProvider = ((defaultSource, processorName, tenantDescriptor, configuration) -> defaultSource);
     }
 
     /**
-     *  Initializes a {@link MultiTenantEventProcessingModule} with a default {@link TenantProvider} and a {@link MultiTenantStreamableMessageSourceProvider},
-     *  which allows for the customization of the {@link StreamableMessageSource} for each {@link TenantDescriptor}.
+     * Initializes a {@link MultiTenantEventProcessingModule} with a default {@link TenantProvider} and a default {@link MultiTenantStreamableMessageSourceProvider},
+     * which does not change the default {@link StreamableMessageSource} for any {@link TenantDescriptor}.
      *
      * @param tenantProvider the default {@link TenantProvider} used to build {@link MultiTenantEventProcessor}s
-     * @param multiTenantStreamableMessageSourceProvider the {@link MultiTenantStreamableMessageSourceProvider} used to customize the {@link StreamableMessageSource} for each {@link TenantDescriptor}
```

Comment:
We miss documentation for the `multiTenantStreamableMessageSourceProvider` param.